student.rb

```diff
@@ -1,14 +1,15 @@
+# Student Class
 require_relative 'person'
 
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(id, classroom, name = "Unknown", age = 0)
+  def initialize(id, classroom, name = 'Unknown', age = 0)
```

Comment:
- [ ] kindly update this as well to not pass in the `id` but pass in the `parent_permission` parameter 
- >* example code 
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```